iocage/lib/LaunchableResource.py

```diff
@@ -21,13 +21,14 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
 # IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-"""The main CLI for ioc."""
 import libzfs
 import typing
 
 import iocage.lib.Config.Jail.File.RCConf
 import iocage.lib.Config.Jail.File.SysctlConf
 import iocage.lib.Resource
+import iocage.lib.LaunchableResourceUpdate
+UpdaterModule = iocage.lib.LaunchableResourceUpdate 
```

Comment:
you finally did it, you blew thru the 79 char limit with a class name

Reply:
🤣